xcube/core/normalize.py

```diff
@@ -726,6 +722,8 @@ def _normalize_dim_order(ds: xr.Dataset) -> xr.Dataset:
                 ds = ds.copy()
                 copy_created = True
             ds[var_name] = var.transpose(*dim_names)
+            if var.encoding:
```

Comment:
Why that condition? Anything can be in `var.encoding` so it will not be empty. On the other hand, the dictionary `var.encoding` always exists.

Reply:
Condition is still wrong.

Reply:
We are still discussing what to put there instead.